src/config.rs

```diff
@@ -134,26 +137,41 @@ impl std::fmt::Display for Config {
 #[cfg(test)]
 mod test {
     use super::*;
-    use std::path::Path;
+    use std::{path::Path, fs};
 
     #[test]
     fn simple() {
         let c: Config = toml::de::from_str(r#"
 user = "somebody"
-directory = "/path/to/somewhere"
+directory = "/etc/laurel/config.toml"
```

Comment:
I think there's a misconception here. directory is meant to be the the "working directory", i.e. the base directory for log files.